Feature Add Oracle driver (#2487)
codeigniter4/CodeIgniter4: user_guide_src/source/changelogs/v4.2.0.rst

    - Added the functions ``csp_script_nonce()`` and ``csp_style_nonce()`` to get nonce attributes
    - See :ref:`content-security-policy` for details.
- New View Decorators allow modifying the generated HTML prior to caching. 
- Added new OCI8 driver for database.

>     - You to access Oracle Database. They support SQL and PL/SQL statements.

Reviewer: ```suggestion
    - It can access Oracle Database and supports SQL and PL/SQL statements.
```
